app/src/main/java/org/mozilla/fenix/ext/BrowserState.kt

```diff
@@ -41,7 +42,7 @@ fun BrowserState.asRecentTabs(): List<RecentTab> {
         } else {
             listOf(selectedNormalTab)
                 .plus(normalTabs.sortedByDescending { it.lastAccess })
-                .minus(lastTabGroup?.tabs ?: emptyList())
+                .filterNot { lastTabGroup?.tabIds?.contains(it?.id) ?: true }
```

Comment:
You want to remove the tabs that are in the last group here. So with `filterNot` you'd need to `?: false`, default to false, no?

Reply:
Good catch.  yes should be `?: false` I will update